docs/connect.html

```diff
@@ -98,6 +99,13 @@
     <li><code>DELETE /connectors/{name}</code> - delete a connector, halting all tasks and deleting its configuration</li>
 </ul>
 
+We also provides a REST API for getting information of connector plugins:
+
+<ul>
+    <li><code>GET /connector-plugins</code><- return a list of connector plugins installed in the Kafka Connect cluster</li>
```

Comment:
Not sure if this needs to be documented or not, but this only checks for connectors on the worker that handles the request. This means you can actually see inconsistent results (and should during a rolling upgrade if you add new connector jars)